ban/commands/export.py

```diff
@@ -12,10 +27,29 @@ def resources(path, **kwargs):
 
     path    path of file where to write resources
     """
-    resources = [models.PostCode, models.Municipality, models.Group,
-                 models.HouseNumber]
-    with Path(path).open(mode='w', encoding='utf-8') as f:
-        for resource in resources:
-            for data in resource.select().serialize({'*': {}}):
-                f.write(dumps(data) + '\n')
-                reporter.notice(resource.__name__, data)
+    resources = [Municipality, PostCode, Group, HouseNumber]
+    for resource in resources:
+        query = QUERIES.get(resource)
+        filename = '{}.ndjson'.format(resource.__name__.lower())
+        with Path(path).joinpath(filename).open(mode='w') as f:
+            print('Exporting to', f.name)
+            query = query.order_by(resource.pk)
+            results = []
+            for result in helpers.batch(process_resource, query,
+                                        chunksize=1000, total=query.count()):
+                results.append(result)
+                if len(results) == 10000:
+                    f.write('\n'.join(results) + '\n')
+                    f.flush()
+                    os.fsync(f.fileno())
```

Comment:
Yeah, I'm not sure what we want to do here.
Basically:
- `f.write` will write to the python buffer
- `f.flush` will transfer the data to the OS buffers (I guess this is where we release memory)
- `os.fsync` will actually write on disk

So I added the `os.fsync` line to be able to live control the size of the file for debug purposes, but I think it's better to actually let the OS decide when to write on disk (we don't really care about the scenario where the server lose power during the export process — in this case we will just rerun the export anyway), so maybe it's better to remove `os.fsync`.
But I'd keep the `flush` call, as I understand this is where python will release memory.

Thoughts?